### PR TITLE
Add graceful shutdown for RisePlayer on Windows

### PR DESCRIPTION
**Pull request by ahmedalsudani**
@fjvallarino @tejohnso please review

### CODE REVIEW

**Comment by fjvallarino** [comment]
:+1:

**Comment by tejohnso** [comment]
@ahmedalsudani Why don't we use powershell to place the graceful shutdown file and then taskkill?  Seems simpler than going through the network.

**Comment by ahmedalsudani** [comment]
@tejohnso how do we know that player is running when we issue taskkill?

**Comment by tejohnso** [comment]
@ahmedalsudani good point.  And what about including the PS1 file as part of the repo instead constructing it as part of the install process?

**Comment by ahmedalsudani** [comment]
@tejohnso that we can do. Is there a reason we don't include the batch files in the repo?

**Comment by tejohnso** [comment]
@ahmedalsudani No reason I can think of.

**Comment by fjvallarino** [comment]
@ahmedalsudani @tejohnso the main reason is the scripts use paths which depend on the user's folder or on OS specific folders which may not always be in the same location. We can use the corresponding environment variables, I guess.

**Comment by ahmedalsudani** [comment]
@tejohnso I experimented with having the scripts in separate files, but it does not always work. Specifically, it doesn't work when we're calling the `stop-player.ps1` script. The path to the script cannot be relative, so we have to use `platform.getInstallDir()` in order to call it.

I could move out `stop-player.ps1` alone. However, I don't see what is gained by moving one script into a separate file and leaving 3.

**Comment by ahmedalsudani** [comment]
@tejohnso are you ok with this PR?

**Comment by tejohnso** [comment]
@ahmedalsudani The path is always "%LOCALAPPDATA%/rvplayer/Installer" on windows and "$HOME/rvplayer/installer" on linux.

**Comment by ahmedalsudani** [comment]
@fjvallarino @tejohnso code is ready for review. Before merging, I need to 
- rebase
- update tests

I chose not to move the uninstallation script out of the js file because it involves deleting the shortcuts, which is the most complicated part. If you don't think it would be a problem, let me know

File: build.js [hunk 1]
```diff
@@ -130,6 +131,7 @@ function createSelfExtractingInstallers() {
     "RunProgram=\"installer.exe\"" + "\n" +
     ";!@InstallEnd@!" + "\n";
 
+    spawnSync("sh", ["-c", "cp -r scripts builds/installer-" + platform]);
```
Comment (tejohnso) [inline build.js]: @ahmedalsudani each argument should be a separate element in the array.   "cp -r .." should be all together.

Comment (ahmedalsudani) [inline build.js]: @tejohnso isn't that the case now?

Comment (tejohnso) [inline build.js]: Yeah, looks like it is.

**Comment by tejohnso** [comment]
@ahmedalsudani This pr should be closed and a new one created against release-branch-7

**Pull request closed without merging**